package.json

```diff
@@ -140,7 +140,7 @@
     "mobxpromise": "^1.3.1",
     "node-sass": "^4.3.0",
     "numeral": "^2.0.6",
-    "oncoprintjs": "^1.0.95",
+    "oncoprintjs": "^1.0.98",
```

Comment:
Thanks for including this part, @alisman! I was out of reach of my laptop when the fix came in.